reporter/html.js

```diff
@@ -713,6 +713,10 @@ QUnit.log( function( details ) {
 				" run with a higher max depth or <a href='" +
 				escapeText( setUrl( { maxDepth: -1 } ) ) + "'>" +
 				"Rerun</a> without max depth.</p></td></tr>";
+		} else {
+			message += "<tr class='test-message'><th>Message: </th><td>" +
```

Comment:
this is consistent with [line 710](https://github.com/jquery/qunit/pull/944/files#diff-5b7e94e2a10b848c4ae4b539388f86aeR710) but I wonder if we should keep this `<th>Message: </th>`.

Anyway, this should be addressed in another patch.